extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/PanacheQuery.java

```diff
@@ -0,0 +1,161 @@
+package io.quarkus.mongodb.panache;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import io.quarkus.panache.common.Page;
+
+/**
+ * <p>
+ * Interface representing an entity query, which abstracts the use of paging, getting the number of results, and
+ * operating on {@link List} or {@link Stream}.
+ * </p>
```

Comment:
And you shouldn't close them, it generates a Javadoc warning.

Reply:
Strange that it generates a warning as closing p is optional in HTML. Javadoc is strange.
Anyway, I remove all of them.